.github/workflows/code-check.yml

```diff
@@ -0,0 +1,26 @@
+name: Check code health
```

Comment:
ref: [workflow in line/liff-inspector](https://github.com/line/liff-inspector/blob/main/.github/workflows/code-check.yml)

This workflow is not currently running in this repository.
However, it is running in my forked repository, and there are no errors.
https://github.com/shogo0421/create-liff-app/actions/runs/11548918674

[github docs (about-workflow-runs-from-public-forks)](https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/approving-workflow-runs-from-public-forks#about-workflow-runs-from-public-forks)

> workflows on pull requests to public repositories from some outside contributors will not run automatically, and might need to be approved first. By default, all first-time contributors require approval to run workflows.